content/integrations/google-cloud-secret-manager.md

```diff
@@ -148,15 +150,57 @@ This can be either your internal CA or a public CA such as [Let's Encrypt](https
    The command uses `--auth=off` because our `root.cert` and `app.cert` certificates are self-signed.
    {{< /admonition >}}
 
+
+   **Containers**
+
+   The following instructions use [Podman](https://podman.io/) to manage the containers.
+   You can also use Docker.
+
+   Modify addresses and file paths as needed for your deployment.
+
+   ```sh {.copy}
+   sudo podman pod create  \
+     -p 9000:9000 -p 9001:9001 -p 7373:7373  \
+     -v ~/minio-kes-gcp/certs:/certs  \
+     -v ~/minio-kes-gcp/minio:/mnt/minio  \
+     -v ~/minio-kes-gcp/config:/etc/default/  \
+     -n minio-kes-gcp
+   
+   sudo podman run -dt  \
+     --cap-add IPC_LOCK  \
+     --name kes-server  \
+     --pod "minio-kes-gcp"  \
+     -e KES_SERVER=https://127.0.0.1:7373  \
+     -e KES_CLIENT_KEY=/certs/kes-server.key  \
+     -e KES_CLIENT_CERT=/certs/kes-server.cert  \
+     quay.io/minio/kes:2024-01-11T13-09-29Z server  \
+       --auth  \
+       --config=/etc/default/kes-config.yaml  \
+   
+   sudo podman run -dt  \
+     --name minio-server  \
+     --pod "minio-kes-gcp"  \
+     -e "MINIO_CONFIG_ENV_FILE=/etc/default/minio"  \
+     quay.io/minio/minio:RELEASE.2024-01-31T20-20-33Z server  \
+       --console-address ":9001"
+   ```
+
+   You can verify the status of the containers using the following command.
+   The command should show three pods, one for hte Pod, one for KES, and one for MinIO.
```

Comment:
```suggestion
   The command should show three pods, one for the Pod, one for KES, and one for MinIO.
```

Reply:
Did a search and caught a couple of those that I fixed. Thanks.